### ISSUE TITLE
RFE: is it possible to start making github releases?🤔

### ISSUE THREAD

**Issue by kloczek** [openSUSE/libsolv#547]
On create github release entry is created email notification to those whom have set in your repo the web UI Watch->Releases.
gh release can contain additional comments (li changelog) or additional assets like release tar balls (by default it contains only assets from git tag) however all those part are not obligatory.
In simplest variant gh release can be empty because subiekt of the sent email contains git tag name.

I'm asking because my automation process uses those email notifications by trying to make preliminary automated upgrades of building packages, which allows saving some time on maintaining packaging procedures.
Probably other people may be interested to be instantly informed about release new version as well.

Documentation and examples of generate gh releases:
https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository
https://cli.github.com/manual/gh_release_upload/
https://github.com/jbms/sphinx-immaterial/pull/282
https://github.com/marketplace/actions/github-release
https://pgjones.dev/blog/trusted-plublishing-2023/
https://github.com/jbms/sphinx-immaterial/issues/281#issuecomment-1700933026
tox target to publish on pypi and make gh release https://github.com/jaraco/skeleton/blob/928e9a86d61d3a660948bcba7689f90216cc8243/tox.ini#L42-L58

**Comment by Conan-Kudo** [comment]
You could just use the release-monitoring as your input for automation? https://release-monitoring.org/project/7908/

**Comment by kloczek** [comment]
They do not offer API interface.
Is it any problem with execute one command to generate gh release with empty comment?

**Comment by Conan-Kudo** [comment]
> They do not offer API interface. 

Yes it does: https://release-monitoring.org/static/docs/api.html

(The link to it was at the bottom of the web page.)

> Is it any problem with execute one command to generate gh release with empty comment?

The thing is, getting everyone to make GitHub releases is not going to scale. Maybe @mlschroe will make GitHub releases, maybe he won't, but you cannot rely on GitHub releases in the first place.

**Comment by kloczek** [comment]
> The thing is, getting everyone to make GitHub releases is not going to scale. Maybe @mlschroe will make GitHub releases, maybe he won't, but you cannot rely on GitHub releases in the first place.

Can you explain how automatic sending email notifications is "not going to scale"? 🤔 
About what kind of scale of what you are talking about? 🤔 
Generate gh release im many cases of the projects it is SINGLE line in gh workflow definition if such workflow is used to tag repo and for example upload dist tar ball.

**Comment by eli-schwartz** [comment]
> The thing is, getting everyone to make GitHub releases is not going to scale. Maybe @mlschroe will make GitHub releases, maybe he won't, but you cannot rely on GitHub releases in the first place.

Don't worry, he's trying to scale it anyway :rofl: and has opened mass bug reports against tons of projects with this exact subject.

Here's another one: https://github.com/mesonbuild/meson-python/issues/484#issuecomment-1709195137

In that one, a couple comments down I recommended release-monitoring.org as well. I also recommended several other possible solutions, but my suggestions were universally ignored in favor of "but I'm going to open bug reports against tons of projects, because I noticed most projects have a github VCS anyways".

Which is very ?????????
